### PR TITLE
New Adapter: Boldwin

### PR DESCRIPTION
**Pull request by Wls-demo**
doc - https://github.com/prebid/prebid.github.io/pull/3921

### CODE REVIEW

**Comment by SyntaxNode** [comment]
This looks a lot like the Compass adapter. Is Boldwin an alias of Compass? If so, we have simpler ways of setting up an alias.

File: static/bidder-info/boldwin.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+maintainer:
+  email: "wls_demo_box@smartyads.com"
```
Comment (VeronikaSolovei9) [inline static/bidder-info/boldwin.yaml:2]: "smatryads.com" - we already have it in Smartyads adapter. Please confirm this email here is correct.

File: static/bidder-params/boldwin.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "type": "string",
+        "minLength": 1,
+        "description": "Endpoint ID"
+      }
+    },
+    "oneOf": [
+      { "required": ["placementId"] },
+      { "required": ["endpointId"] }
+    ]
+  }
```
Comment (VeronikaSolovei9) [inline static/bidder-params/boldwin.json:23]: Yes, this is a copy of Compass.json. Also Boldwin bidder code is identical to Compass adapter code, imp.ext is the same and tests are the same. Like @SyntaxNode mentioned already - should this be an alias?

Comment (Wls-demo) [reply static/bidder-params/boldwin.json:23]: This is not an alias, this is a separate adapter

File: static/bidder-info/boldwin.yaml (hunk 1 above)
Comment (Wls-demo) [reply static/bidder-info/boldwin.yaml:2]: this email is correct

**Comment by bsardo** [comment]
Hi @Wls-demo, Thanks for your contribution. We're curious why you've copied the entire Compass adapter including config files and tests. We strongly prefer avoiding duplicate adapter code.

**Comment by Wls-demo** [comment]
@bsardo I do apologise but this implementation is aligned with platform's needs. As we have list of partners who want to work with us, I would kindly ask you to proceed as it is.

**Comment by VeronikaSolovei9** [comment]
Local testing looks good. Ready to approve it if team will be ok with duplicated code

**Comment by bsardo** [comment]
@Wls-demo looks good. Could you please update the docs to indicate that either `placementId` or `endpointId` must be provided since your bidder-params json file specifies that one of them is required?

**Comment by Wls-demo** [comment]
@bsardo thanks. We updated docs

**Review by bsardo — APPROVED** [review]

**Review by VeronikaSolovei9 — APPROVED** [review]

**Pull request merged**